internal/api/list_analysistemplates_v1alpha1.go

```diff
@@ -40,8 +42,14 @@ func (s *server) ListAnalysisTemplates(
 		return nil, fmt.Errorf("list analysistemplates: %w", err)
 	}
 
-	sort.Slice(list.Items, func(i, j int) bool {
-		return list.Items[i].Name < list.Items[j].Name
+	slices.SortFunc(list.Items, func (a, b rollouts.AnalysisTemplate) int {
+		if a.Name < b.Name {
+			return -1
+		} else if a.Name > b.Name {
+			return 1
+		} else {
+			return 0
+		}
```

Comment:
This can be done much more efficiently.

```suggestion
		strings.Compare(a.Name, b.Name)
```

It seems a similar change is needed in a number of other spots as well.